ui/packages/form/src/Dropdown.svelte

```diff
@@ -7,13 +7,14 @@
 	export let choices: Array<string>;
 	export let style: string = "";
 	export let disabled: boolean = false;
+	export let form_position: "first" | "last" | "mid" | "single" = "single";
 
 	const dispatch = createEventDispatcher<{ change: string }>();
 
 	$: dispatch("change", value);
 </script>
 
-<Block>
+<Block {form_position}>
```

Comment:
We might be able to move this Block wrapping to the app and prevent some repetition by doing it in the `each` when we render the components. It would clean up the components quite a bit.

Reply:
#1055